re2c/src/util/u32lim.h

```diff
@@ -23,6 +23,9 @@ class u32lim_t
 	explicit u32lim_t (uint64_t x)
 		: value (x < LIMIT ? static_cast<uint32_t> (x) : LIMIT)
 	{}
+	explicit u32lim_t (size_t x)
```

Comment:
Added constructor for `size_t` because on OS X `size_t` is neither `uint32_t` nor `uint64_t`.